tasks/sast-java-sec-check.yaml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sast-java-sec-check
+spec:
+  workspaces:
+    - name: workspace
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
+  params:
+    - name: PATH_CONTEXT
+      description: The path to your source code
+      type: string
+      default: .
+    - name: OUTPUT_FORMAT
+      description: the format of findsecbugs output
+      type: string
+      default: "-sarif"
+    - name: OUTPUT_ONLYANALYZE
+      description: |
+        only analyze given classes and packages;
+        end with .* to indicate classes in a package, .- to indicate a package prefix
+      type: string
+      default: ""
+    - name: OTHER_PARAM
```

Comment:
I think this could be named `OPTIONAL_ARGS` or `OPTIONAL_FLAGS` for the command to communicate the intention better.